### ISSUE TITLE
Disable log for some query

### ISSUE THREAD

**Issue by wenerme** [mikro-orm/mikro-orm#4223]
**Is your feature request related to a problem? Please describe.**

By using sequelize I can disable log like

```ts
const user = await User.findOne({  
  logging: false,
  where: { id: 2207 }
})
```

When upsert or insert many rows, this is very helpful, I want to disable some log when using mikro orm.

**Describe the solution you'd like**

Maybe add or logging to the second option ?

**Describe alternatives you've considered**

als control the Logger

**Additional context**

**Comment by B4nan** [comment]
cc @SpencerKaiser, since you were already working with the query logging, this one could be a simple task for you? :]

**Comment by SpencerKaiser** [comment]
@B4nan yea that'd work out nicely as an addition after the logger context stuff gets added in! I'll take a look as soon the other is completed and merged 🙂

**Comment by B4nan** [comment]
Would be great to make this work both ways - `logging: true` would enforce logging even if it's globally disabled, while `logging: false` would disable it when it's globally enabled.

**Comment by B4nan** [comment]
Implemented in v6 via #4273
